src/main/java/model/LottoNum.java

```diff
@@ -0,0 +1,50 @@
+package model;
+
+import valueObject.ErrorMessage;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class LottoNum {
+    private static final int LOTTO_PRICE = 1000;
+    private static final int LOTTO_SIZE = 6;
+
+    private ErrorMessage errorMessage = new ErrorMessage();
+    // 금액을 입력받고 몇 장을 샀는지 확인하는 메서드
+    public int buyLotto(int money) {
+        if (money%1000 != 0) {
+            throw new IllegalArgumentException(errorMessage.ERROR_PRICE);
+        }
+        return money/LOTTO_PRICE;
+    }
+
+    // 랜덤으로 숫자를 뽑아 리스트에 넣어주는 메서드
+    public List<Integer> lottoNumbers() {
+        List<Integer> randNum = new ArrayList<Integer>();
```

Comment:
줄임말을 사용하는 것을 자제하고 구체적으로 변수명을 짓는 것이 좋습니다.
```suggestion
        List<Integer> randomNumbers = new ArrayList<Integer>();
```